packages/sources/terra-view-function/src/config/index.ts

```diff
@@ -35,13 +36,19 @@ const buildRpcUrlMapping = () => {
   for (const chainId of SUPPORTED_CHAIN_IDS) {
     // Underscore-ize and capitalize to format for environment variables
     const prefix = chainId.replace('-', '_').toUpperCase()
-    const envVar = util.getEnv(ENV_RPC_URL, prefix)
+    const envVar = util.getEnv(ENV_LCD_URL, prefix)
+    const legacyEnvVar = util.getEnv(ENV_RPC_URL, prefix)
     if (envVar) {
       output[chainId] = envVar
       hasAtLeastOneURL = true
+    } else if (legacyEnvVar) {
+      console.log(
+        'The RPC_URL environment variable is being deprecated, please use an LCD_URL instead. See README for more information.',
+      )
+      output[chainId] = legacyEnvVar
+      hasAtLeastOneURL = true
     }
   }
-
   if (!hasAtLeastOneURL) throw new Error('At least one RPC URL must be defined')
```

Comment:
We should also update this error message to "At least one LCD URL must be defined"

Reply:
Done, also there were some other variables I missed using rpc URL classifications that I fixed